multi-project/core/src/main/java/by/artempvn/lab01/core/Utils.java

```diff
@@ -0,0 +1,20 @@
+package by.artempvn.lab01.core;
+
+import by.artempvn.lab01.util.StringUtils;
+
+public class Utils {
+
+	public static boolean isAllPositiveNumbers(String... str) {
+		if (str == null) {
+			return false;
+		}
+		int i = 0;
+		boolean isAllPositive = true;
+		while (i < str.length && isAllPositive) {
+			isAllPositive = StringUtils.isPositiveNumber(str[i]);
+			i++;
+		}
+		return isAllPositive;
```

Comment:
Do you familiar with Stream API? If not, you can learn how to use it watching this video: [Get a Taste of Lambdas and Get Addicted to Streams by Venkat Subramaniam](https://www.youtube.com/watch?v=1OpAgZvYXLQ).
Please refactor this method using Stream API.

Reply:
Yes, just didn't think about using it hear, I have little expirience with streams.